extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -177,7 +181,7 @@ export function getElement(ampdoc, selector, analyticsEl, selectionMethod) {
     // Only tag names are supported currently.
     foundEl = closestByTag(analyticsEl, selector);
   } else if (selectionMethod == 'scope') {
-    foundEl = analyticsEl.parentElement.querySelector(selector);
+    foundEl = scopedQuerySelector(analyticsEl.parentElement, selector);
```

Comment:
There's the same thing in the `analytics-root`. This code is actually waiting to be killed in preference of that. Can you change there as well?